src/js/components/PackageUpgradeOverview.js

```diff
@@ -1,135 +0,0 @@
-import React from 'react';
```

Comment:
This is now going to be implemented in the feature branch `feature-branch/software-updates`, so this component shouldn't be here.